player/video.c

```diff
@@ -1247,7 +1248,7 @@ void write_video(struct MPContext *mpctx)
     struct vo_frame *frame = vo_frame_ref(&dummy);
 
     double diff = mpctx->past_frames[0].approx_duration;
-    if (opts->untimed || vo->driver->untimed)
+    if (opts->untimed || mpctx->video_out->driver->caps & VO_CAP_UNTIMED)
```

Comment:
```suggestion
    if (opts->untimed || vo->driver->caps & VO_CAP_UNTIMED)
```

Reply:
also its fine, but IIRC we generally do `(a & b)` for readability instead `a & b`. But you decide. Unnecessary parentheses are optional ;p But even in this PR two styles are mixed.